src/Components/MessageState/EmptyStates.js

```diff
@@ -87,4 +90,34 @@ const NoMatchingRecs = () => {
   );
 };
 
-export { ErrorState, NoAffectedClusters, NoMatchingClusters, NoMatchingRecs };
+// used in Routes.js to create custom message instead of <InvalidObject>
+const ComingSoon = () => {
+  const intl = useIntl();
+  return (
+    <EmptyState variant="small">
+      <Spinner />
```

Comment:
There is no spinner at the mock-ups. I suppose it should be the `pf-icon-in-progress` instead.